tests/pkg/tenant/tenant_setup.go

```diff
@@ -272,47 +249,3 @@ func (mgr TestTenantsManager) List() []Tenant {
 
 	return toReturn
 }
-
-func (mgr TestTenantsManager) cleanup(ids []string) {
```

Comment:
Now as we have removed DB access logic from tests, we can remove their DB access entirely by removing:

- The cloudSQL proxy from the testDefinitions from charts.
- The DB credentials and connection details from their env.
- Their service accounts from the proxy-user workload identity in our infra projects.